scripts/sshzobelisk

```diff
@@ -0,0 +1,40 @@
+set -euo pipefail
+
+usage()
+{
+    echo "USAGE: $0 <USERNAME ON ZOBELISK> <USER ID ON ZOBELSIK>"
+    exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+LOCAL_USER=$USER
+SERVER_USERNAME=$1
+SERVER_UID=$2
+LOCAL_REPO=$(realpath $(dirname $BASH_SOURCE)/../)
+ZOBELISK_IP=192.168.37.176
+IP=`hostname -I`
+IP=`echo $IP | head -n1 | awk '{print $1;}'` # `echo $IP | sed 's/ *$//g'`
+echo $IP
+
+sudo systemctl start ssh
+sleep 1
+
+ROS_PORT=${SERVER_UID}0
+
+export ROS_MASTER_URI=http://$ZOBELISK_IP:$ROS_PORT/
+
+tmux new-session -d
+
+ssh $SERVER_USERNAME@$ZOBELISK_IP \
+    -t \
+    "export ROS_MASTER_URI=http://$ZOBELISK_IP:$ROS_PORT && \
+    sshfs -o nonempty -o allow_other \
```

Comment:
Why are you doing the sshfs backwards? Why not just sshfs from the user's machine to zobelisk? This way you can avoid the whole process of getting the $IP

Reply:
so to the best of my knowledge, this is just how it works, the only better alternative is to run a ftps server, but that is alot.

Reply:
sshfs can go both ways. It makes more sense to run `sshfs <local workspace> <zobelisk>:<zobelisk workspace>`. This way the user doesn't have to run an ssh server.

Reply:
Also, I'm worried that this would wipe data if the user already had a catkin_ws on zobelisk. What would happen in this case?

Reply:
> sshfs can go both ways. It makes more sense to run `sshfs <local workspace> <zobelisk>:<zobelisk workspace>` from the user's machine. This way the user doesn't have to run an ssh server.

this is not how sshfs works https://superuser.com/questions/616182/how-to-mount-local-directory-to-remote-like-sshfs

Reply:
> Also, I'm worried that this would wipe data if the user already had a catkin_ws on zobelisk. What would happen in this case?

Data is not lost.
DEMO:
2 bash session, A and B on user's computer

in bash session A
`ssh <username>@192.168.37.176`
`echo "test" > ~/catkin_ws/src/mil/test.txt`
`cat ~/catkin_ws/src/mil/test.txt`

in bash session B,
`~/catkin_ws/src/mil/scripts/sshzobelisk <username> <user_id>`
`mil`
`ls`

In bash session A
`cat ~/catkin_ws/src/mil/test.txt`
The data will appear to be lost

In bash session B
`exit`
`exit`

In bash session A
`cat ~/catkin_ws/src/mil/test.txt`
and see that The data is not lost